rts/Game/InMapDrawModel.cpp

```diff
@@ -161,24 +161,28 @@ void CInMapDrawModel::EraseNear(const float3& constPos, int playerID)
 		for (int x = xStart; x <= xEnd; ++x) {
 			DrawQuad* dq = &drawQuads[(y * drawQuadsX) + x];
 
-			for (auto pi = dq->points.begin(); pi != dq->points.end(); /* none */) {
+			//for (auto pi = dq->points.begin(); pi != dq->points.end(); /* none */) {
```

Comment:
No point in commenting out the old loops if you don't leave behind a comment explaining why you chose explicit indexing, tell the reader MSVC chokes on iterator manipulation.

Reply:
Deal, will be fixed.